.github/workflows/ci_pipeline.yml

```diff
@@ -169,9 +170,10 @@ jobs:
           git rm -rf .
 
       - name: Download documentation artifacts
-        uses: actions/download-artifact@v3
+        uses: actions/download-artifact@v4
         with:
           name: docs
+          merge-multiple: true
```

Comment:
I believe this is not actually needed.

Reply:
maybe the ide here is to avoid an extra dir in the intermediate path?

Reply:
That makes sense if you have multiple jobs generating an artifact, but since we have a single one, it's probably not required. See https://github.com/actions/upload-artifact/blob/main/docs/MIGRATION.md#multiple-uploads-to-the-same-named-artifact

Reply:
Indeed, I agree it may not be necessary, just what whoever wrote this preferred.

Since this might change paths and with a small probability have unintended effects in the deploy process later (haven't checked), but has no other advantages to fix otherwise, I vote for keeping as is (it-aint-broken policy).

Reply:
+1 to keeping it. Eventually, we'll need to version documentation. Sooner or later we'll wind up having to collate generated documentation artifacts.